### PR TITLE
Feat/develop/payment integration

### PR DESCRIPTION
**Pull request by sakthi-ionixx**
ValidPaymentLimit API integration

### CODE REVIEW

File: app/src/lib/providers/PaymentProvider.tsx [hunk 1]
```diff
@@ -27,100 +32,151 @@ export interface PaymentData {
   sessionKey?: string;
 }
 
+export interface ShowPaymentMethods {
```
Comment (alexinx) [inline app/src/lib/providers/PaymentProvider.tsx:35]:
```suggestion
export interface PaymentMethodLimit {
```

File: app/src/lib/providers/PaymentProvider.tsx [hunk 2]
```diff
@@ -27,100 +32,151 @@ export interface PaymentData {
...
+  showCreditCard?: boolean;
```
Comment (alexinx) [inline app/src/lib/providers/PaymentProvider.tsx:36]:
```suggestion
  exceedCreditCard?: boolean;
```

File: app/src/stories/creditCard.stories.tsx [hunk 3]
```diff
@@ -32,7 +32,11 @@ export const CreditCardStories = () => {
         wire: true,
         creditCard: true,
       }}
-      buttonDisabled={ false } />
+      buttonDisabled={ false }
+      PaymentMethodLimit={{
```
Comment (ShowriSrinivas) [inline app/src/stories/creditCard.stories.tsx:36]: Prop name is wrong change it to paymentMethodLimit

File: app/src/stories/wireTransfer.stories.tsx [hunk 4]
```diff
@@ -32,7 +32,11 @@ export const WireTransferStories = () => {
         wire: true,
         creditCard: true,
       }}
-      buttonDisabled={ false } />
+      buttonDisabled={ false }
+      PaymentMethodLimit={{
```
Comment (ShowriSrinivas) [inline app/src/stories/wireTransfer.stories.tsx:36]: same as above

**Review by ShowriSrinivas — CHANGES_REQUESTED** [review]

**Review by ShowriSrinivas — APPROVED** [review]

**Pull request merged**